neo/SmartContract/InteropService.cs

```diff
@@ -435,6 +442,25 @@ private static bool Block_GetTransactionCount(ApplicationEngine engine)
             return false;
         }
 
+        private static StackItem TransactionToStackItem(Transaction tx)
```

Comment:
Maybe we should create a new interface:

```c#
namespace Neo.SmartContract
{
    public interface IInteroperable
    {
        StackItem ToStackItem();
    }
}
```

We can do this in another PR first.

Reply:
Yes, Much better

Reply:
We need to move blocks too, so if you prefer in other, or in this, both options are good for me

Reply:
I thought it again. We can do it in this PR.